params/config.go

```diff
@@ -91,6 +93,7 @@ var (
 		EIP158Block:         big.NewInt(3),
 		ByzantiumBlock:      big.NewInt(1035301),
 		ConstantinopleBlock: big.NewInt(3660663),
+		PetersburgBlock:     big.NewInt(9999999), //TODO! Insert Rinkeby block number
```

Comment:
What about `3800083`?

Reply:
Now we're at `3729555`, `3800083` is in abput `11.4` days from now. @karalabe what's your take?